quickstart/101-synapse/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
```

Comment:
Please move the terraform and provider blocks into a providers.tf file.

Reply:
@grayzu done